src/views/activity/Activity.vue

```diff
@@ -65,12 +65,12 @@ div
     div.row
       div.col-lg-6
         b-form-checkbox(v-model="filterAFK")
-          | Filter AFK
+          | Exclude time away from computer
         b-form-checkbox(v-model="includeAudible")
-          | Count audible browser tab as active
+          | Always treat browsers playing audio or video as active
```

Comment:
This might be a better description, but the long line is an eyesore. 'video' could definitely be dropped here, probably 'always' too.

After some thought, I prefer the old phrasing though.